05-advanced-projects/01-AWS-architecture-diagram-generator/.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+.yarn/install-state.gz
+
+# testing
+/coverage
+
+# next.js
+/.next/
+/out/
+
+# production
+/build
+
+# misc
+.DS_Store
```

Comment:
It seems like the DS_Store files are not ignored because there are already ones in the repo. You need to ignore them all.